tmail-backend/tmail-third-party/ai-bot/src/main/java/com/linagora/tmail/conf/AIBotProperties.java

```diff
@@ -0,0 +1,45 @@
+package com.linagora.tmail.conf;
+
+
+import java.net.URL;
+import java.util.Optional;
+
+public class AIBotProperties {
```

Comment:
IMO this class is a pure duplicate of AIBotConfig

Remove it and favor a `public static AIBotConfig fromConfiguration(PropertiesProvider propertiesProvider)` method